doc/api/stream.md

```diff
@@ -1197,7 +1197,9 @@ print(fs.createReadStream('file')).catch(console.log);
 
 If the loop terminates with a `break` or a `throw`, the stream will be
 destroyed. In other terms, iterating over a stream will consume the stream
-fully.
+fully. 
+Note that by default file iteration would be by the chunks of the size equal
+to [`highWaterMark`][] value(default is 16kb).
```

Comment:
There appears to be no link for `highWaterMark` so this does not render as a link.

Reply:
Please put a space between `value` and `(`.

Reply:
fixed